modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -327,6 +327,53 @@ def ip(self):
             self._ip_cache = ray.get(self._ip_cache)
         return self._ip_cache
 
+    def split(self, split_func, num_splits, *args):
+        """
+        Split the object wrapped by the partition into multiple partitions.
+
+        Parameters
+        ----------
+        split_func : Callable[pandas.DataFrame, List[Any]] -> List[pandas.DataFrame]
+            The function that will split this partition into multiple partitions.
+        num_splits : int
+            The number of resulting partitions (may be empty).
+        *args : List[Any]
+            Arguments to pass to ``split_func``
+
+        Returns
+        -------
+        list
+            A list of partitions.
+        """
+        logger = get_logger()
+        logger.debug(f"ENTER::Partition.split::{self._identity}")
+
+        @ray.remote(num_returns=num_splits)
+        def _split_df(df, split_func, *args):  # pragma: no cover
+            return split_func(df, *args)
+
+        outputs = _split_df.remote(self._data, split_func, *args)
+        logger.debug(f"SUBMIT::_split_df::{self._identity}")
```

Comment:
should this be above the `_split_df.remote` call?

Reply:
Good question - I followed the syntax for `apply`, and just noticed that in the `apply_list_of_funcs` case it appears before, but in the `apply` single func case it appears after. Ultimately, since the `remote` call is just a queuing call it shouldn't super matter, but it would be good to have a standard for this. @naren-ponder - thoughts?